src/nrc/conf/base.py

```diff
@@ -5,114 +5,111 @@
 from .api import *  # noqa
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-DJANGO_PROJECT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir))
-BASE_DIR = os.path.abspath(os.path.join(DJANGO_PROJECT_DIR, os.path.pardir, os.path.pardir))
+DJANGO_PROJECT_DIR = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), os.path.pardir)
+)
+BASE_DIR = os.path.abspath(
+    os.path.join(DJANGO_PROJECT_DIR, os.path.pardir, os.path.pardir)
+)
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv('SECRET_KEY')
+SECRET_KEY = os.getenv("SECRET_KEY")
```

Comment:
I'd prefer to see the `config` helper defined like in Open-Zaak and using that. There shouldn't be a single `os.getenv` left in the settings modules.

Reply:
Actually, this module should be deleted completely :)